packages/code-display/src/CodeDisplay.tsx

```diff
@@ -112,4 +123,22 @@ export const CodeDisplay: FC<PrismEditorProps> = ({
       )}
     </Highlight>
   )
+  if (!onChange) {
+    return highlight(code)
+  }
+  return (
+    <Editor
```

Comment:
I think we should have separate components for Display and Editing. I'm presuming `Editor` is heavier than `Highlight`